.buildkite/nightly-benchmarks/README.md

```diff
@@ -3,30 +3,51 @@
 
 ## Introduction
 
-This directory contains the performance benchmarking CI for vllm.
-The goal is to help developers know the impact of their PRs on the performance of vllm.
+This directory contains two sets of benchmark for vllm.
+- Performance benchmark: benchmark vllm's performance under various workload, for **developers** to gain clarity on whether their PR improves/degrades vllm's performance
+- Nightly benchmark: compare vllm's performance against alternatives (tgi, trt-llm and lmdeploy), for **the public** to know when to choose vllm.
 
-This benchmark will be *triggered* upon:
+
+See  [vLLM performance dashboard](https://perf.vllm.ai) for the latest performance benchmark results and [vLLM GitHub README](https://github.com/vllm-project/vllm/blob/main/README.md) for latest nightly benchmark results.
+
+
+## Performance benchmark quick overview
+
+**Benchmarking Coverage**: latency, throughput and fix-qps serving on A100 (the support for FP8 benchmark on H100 is coming!), with different models.
```

Comment:
> the support for FP8 benchmark on H100 is coming!

For frameworks that currently do not support FP8, how will they be handled? Will Int8 be used as a substitute or left empty?

Reply:
The serving engines that I am benchmarking (vllm, lmdeploy, tgi and trt) all support fp8 now

Reply:
> The serving engines that I am benchmarking (vllm, lmdeploy, tgi and trt) all support fp8 now

LMDeploy TurboMind currently supports quantization for AWQ(W4A16), KV Cache Int4, and KV Cache Int8. FP8 is not supported yet but the feature is under way. We expect to first complete the optimization of GEMM(faster than cuBLAS at small batch sizes) and support for MOE. Targeted optimization for the Hopper architecture is also being planned, and it is expected to be faster than Flash Attention 3. The support for the new features of FP8 and Hopper has not been done before because we do not have an H100 development environment. It is highly likely that cloud servers will be used to support it in the future. I highly appreciate the work of W8A8(FP8) in vLLM and the optimization work of MOE, which is very worthy for us to learn. Cheers.

Reply:
I will start developing fp8 benchmarks after LMDeploy supports it then.